### DISCUSSION TITLE
The Add Action menu has unclearly-named items organized alphabetically [new user observation 2 of 7]

### DISCUSSION THREAD

**Discussion by davidbbs** [home-assistant/frontend#17227] [Automations / scripts / scenes]
I observed a new user creating simple light automations for three hours this weekend. 
Observation 2 of 7:

**Problem 2: The Add Action menu has unclearly-named items organized alphabetically
**Proposal 2: Clarify the names, re-order by common usage, and group by function****
_(The new user would not have figured this out without my help)_

Here is a simple problem the new user ran into: To figure out how to turn a light on or off you have to know that the item is “Call service” and then you search for a “service” called “light.turn_on” or “home assistant.turn_on”. This is not intuitive unless you are used to writing YAML and using “service: “ in your code. 

Further, the items are sorted alphabetically, which means that you have to look through the whole list to find items by function and the very uncommonly used by beginners “Condition” is the third item they see. For example, choose, if-then, and repeat are all control-flow items, so they should be together so you can see the options for control flow, while call service, scene, and play media are all actions that should be together.

In addition, “device” is unclear: what does “device” do? When I select it, I see that it provides common actions for most devices. This is probably the most useful item in the list for new users, but they will only discover it by accident since it is the 5th one down and not clear what it means.

Here is an alternative organization that will make the menu easier for new users by 1) clarifying the names, 2) putting the most common ones at top, and 3) grouping the items by what they do. In particular, this provides 4 example services so that users can see what the most common services look like and see how to explore other services.

- Common Device Actions    _<— (renamed from “Device” to say what it does)_
- Activate Scene
- ======= 
- Simple Turn On    _<— (inserts a homeassistant.turn_on service call)_
- Simple Turn Off    _<— (inserts a homeassistant.turn_off service call)_
- Simple Toggle      _<— (inserts a homeassistant.toggle service call)_
- Advanced Light Turn On (color/brightness)    _<— (inserts a light.turn_on service call)_
- Play Media
- Other Actions…      _<— (inserts a service call, but renamed to make it clear what that is)_
- ======= 
- If-then
- Repeat
- Choose (multiple If-then)   _<— (explain what this does in the title)_
- Stop Condition    _<— (explain what “Condition” really does)_
- =======      _<— (special/advanced items at the bottom)_
- Generate Event
- Define Variable
- Run in parallel
- Wait for a template
- Wait for a trigger

**Comment by matthiasdebaat** [comment]
Thanks for this feedback! Can you tell me a bit more about the user, for example home automation experience, long term goal, etc.?What are the Home Assistant permissions settings, for example is advanced mode on?
